code/sha3/Hacl.Spec.SHA3.Equiv.fst

```diff
@@ -849,19 +864,23 @@ let storeState_inner_unroll_sub_12_18 #m s i =
   let b24 = Spec.storeState_inner s 24 b23 in
   let block = storeState_inner_unroll s in
   //s.[12]
```

Comment:
don't know what rlimit you're running with here, but FYI I wouldn't find it shocking if the default value for the whole file was to be 100